tests/basic/storage_pool_test.go

```diff
@@ -9761,3 +9761,190 @@ var _ = Describe("{AddDriveMetadataPool}", func() {
 		AfterEachTest(contexts)
 	})
 })
+
+var _ = Describe("{AddDriveBeyondMaxSupported}", func() {
+
+	/*
+		Add Drive using legacy add drive feature
+		test max drive per pool
+		test max pool per node
+		test max drives per node
+	*/
+	var testrailID = 19170501
+	// testrailID corresponds to: https://portworx.testrail.net/index.php?/tests/view/19170501
+	var runID int
+	JustBeforeEach(func() {
+		StartTorpedoTest("AddDriveBeyondMaxSupported", "Initiate pool Cloud add-drive beyond permitted and expect error", nil, testrailID)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+	var contexts []*scheduler.Context
+
+	stepLog := "Test max limit of pool,node and max pool in a cluster"
+	It(stepLog, func() {
+		log.InfoD(stepLog)
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("pooladddrivebeyondmax-%d", i))...)
+		}
+		ValidateApplications(contexts)
+		defer appsValidateAndDestroy(contexts)
+		stNodes := node.GetStorageNodes()
+		if len(stNodes) == 0 {
+			dash.VerifyFatal(len(stNodes) > 0, true, "Storage nodes found?")
+		}
```

Comment:
do we need 2 conditions to be verified here ?

1) if len(stNodes) == 0 
2) if len(stNodes) > 0. 

code flow will never reach dash.VerifyFatal until you have stNodes == 0 , 

better to remove if len(stNodes) == 0 {

Reply:
done made changes